docker-compose.yml

```diff
@@ -5,9 +5,9 @@ services:
     image: postgres:16-alpine
     restart: unless-stopped
     environment:
-      - POSTGRES_DB=${DB_NAME}
-      - POSTGRES_USER=${DB_USER}
-      - POSTGRES_PASSWORD=${DB_PASSWORD}
+      - POSTGRES_DB=${DB_NAME:-master}
+      - POSTGRES_USER=postgres
```

Comment:
I'm not getting why not use `- POSTGRES_USER=${POSTGRES_USER:-postgres}` in this case?

Reply:
When a new database is created in the container from scratch, it has default access credentials. For Postgres databases, they are as follows:
user: postgres
password: postgres

If users set other credentials in the .env file (which typically takes place when they frequently run the app in a container and without a container), connection to the database in the container fails because of invalid credentials. What I suggest here is to ignore credentials from the user's .env file and always use the correct default credentials when the app runs in a container. If we use default values for credential variables (`POSTGRES_USER=${DB_USER:-postgres}`), it will work only if the appropriate variables have empty values in the .env file. If they are not empty, connection to the database in the container will fail, so, we'll have the same issue as we had previously.

Reply:
this compose file could be in use by oracle holders, node operators, council daemon holders. We allowed to configu user for database and set their password (look at env.validation file). So i against of changing config in docker-compose

Reply:
This variant looks fine: POSTGRES_USER=${DB_USER:-postgres}, but default values should not be left in the Docker Compose file. The user should correctly set DB_USER and DB_PASSWORD for Kapi to connect to the database